articles/defender-for-cloud/devops-faq.md

```diff
@@ -164,6 +165,14 @@ We don’t support automatic migration for the Defender for DevOps connectors fr
 
 If you want to move a connector’s location, for example a GitHub or Azure DevOps connector, to be stored in a different region than the original one where the connector was created, the recommendation is to delete the existing connector and then to create another connector in the new region.
 
+### How can I see the MSDO scan results in GitHub
+
+Follow the guidance to set up the GitHUb Action [here](https://learn.microsoft.com/en-us/azure/defender-for-cloud/github-action).
+
+Ensure that you have Read and Write permissions set on the workflow permissions in the GitHub repository. See more [here](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository#configuring-the-default-github_token-permissions).
+
+![image](https://github.com/MicrosoftDocs/azure-docs/assets/6423037/90fe1da6-2d41-4cf0-b93c-e69d1a348aff)
```

Comment:
this image needs to be saved and added to the media folder and in this format with alt text 
![image](https://github.com/MicrosoftDocs/azure-docs/assets/58463413/c3a53b50-6a0a-4f13-9d25-d31edfe04562)